src/js/vue3.js

```diff
@@ -0,0 +1,10 @@
+import route from './index.js';
+
+export const ZiggyVue3 = {
+    install: (app, options) => {
+        const _route = (name, params, absolute, config = options) => route(name, params, absolute, options);
+
+        app.config.globalProperties.route = _route;
```

Comment:
What exactly does this line do? Adding it to `globalProperties` specifically.

Reply:
Without this line, a user would have to have to inject `route()` into every component, even if they don't need it in the `setup()` function. This allows `route()` to be used in templates globally.

The below would have to be in every Vue component that wants to use `route()`.

```javascript
<script>
import { inject } from 'vue';
export default {
    setup() {
        const route = inject('route');
        return { route };
    },
};
</script>
```

Reply:
Serving as the `mixin` to make the function globally available to Vue.

Reply:
Do you know if the `globalProperties` thing works _exactly_ the same as mixins? Can you still call `this.route()` inside components using the Options API?

Reply:
@bakerkretzmar What do you mean by "inside components" do you mean in the template or in the script? Because if it's in the script the composition API doesn't have a `this` that's why mixins don't work in the composition API and we need a `useRoute` method

Reply:
Yeah I just meant using `this.route()` inside the script part of a component using the Options API.